### PR TITLE
Bundle approve and request in a single action

### LINKED ISSUE #6430: Bundle approve with the subsidised attestations request in KomenciKit
### Expected Behavior

A komenciKit consumer should not have to issue the `cUSD.approve` for the meta transaction wallet as a separate meta transaction, but have that bundled automatically in the `requestSubsidisedAttestations` action.

See https://github.com/celo-org/komenci/pull/153 for more details. 

With the new API, KomenciKit can pass in the `cUSD.approve` meta transactions in the background thus deprecating the `approveAttestations`, which won't have to be called separately anymore.

### Current Behavior

Consumers of the library need to explicitly execute `approveAttestations` before `requestAttestations`

### PR DESCRIPTION
**Pull request by bowd**
### Description

Implement the changes described in #6430 which removes the need to execute `cUSD.approve` as a separate transaction.
These changes are dependent on https://github.com/celo-org/komenci/pull/153 to be deployed in the API the client is pointing to. 

### Tested

Executed flow locally using the `attestation-flow` utility

### Related issues

- Fixes #6430 

### Backwards compatibility

Changes are _not_ backwards compatible.

### CODE REVIEW

File: packages/komencikit/src/kit.ts [hunk 1]
```diff
@@ -2,11 +2,11 @@ import { Address, normalizeAddressWith0x, serializeSignature, sleep } from '@cel
 import { Err, Ok, Result } from '@celo/base/lib/result'
 import { CeloTransactionObject, CeloTxReceipt } from '@celo/connect'
 import { ContractKit } from '@celo/contractkit'
-import { ContractEventLog } from '@celo/contractkit/lib/generated/types'
```
Comment (bowd) [inline packages/komencikit/src/kit.ts]: @gastonponti maybe you have more insight on this, but I think e might have made some changes to the `tsconfig` and the generated type files (`.d.ts`) aren't copied to `lib` when the package is built, so I had to change this import to the `src` folder. 

I don't think it's a problem in this case but I wanted to make sure that we know this is happening and it's intended behaviour.

**Review by tarikbellamine — COMMENTED** [review]
Can we also get `escrow.withdraw` and `cUSD.transfer` bundled into one transaction? This will make the escrow redemption process more efficient as well

File: packages/komencikit/src/kit.ts (hunk 1 above)
Comment (gastonponti) [reply packages/komencikit/src/kit.ts]: Mmmm, are you using a fixed version of the contractkit?
This was part of a problem that I'm pretty sure that was fixed. We forgot to add those types in the `@celo/typechain-target-web3-v1-celo` package (that the version `-beta3` of that typechain should be the one used in protocol right now in master), and was the one used to release the `contractkit-beta4`

If you are using a fixed version, use the -beta4 one (soon the 1.0.0, but still on testing)
@bowd

**Review by tarikbellamine — APPROVED** [review]
If you've addressed @gastonponti's comment, then the rest looks good to me!

**Pull request merged**